### ISSUE TITLE
Lots of output (5 second chunks) and slow download from Youtube

### ISSUE THREAD

**Issue by alex4200** [yt-dlp/yt-dlp#1177]
## Checklist

- [x] I'm asking a question
- [x] I've looked through the README and FAQ for similar questions
- [x] I've searched the bugtracker for similar questions including closed ones

## Question

I am using `yt-dlp` (currect version 2021.9.25) to download a youtube playlist, but I am not sure if it works correctly. The syntax I am using is 

`yt-dlp --playlist-start 8 --yes-playlist -i -x --audio-format mp3 https://...`

And I get blocks of outputs like these:

```
...
[hls @ 0x7fe920000000] Skip ('#EXT-X-VERSION:3')
[hls @ 0x7fe920000000] Skip ('#EXT-X-DISCONTINUITY-SEQUENCE:278')
[hls @ 0x7fe920000000] Skip ('#EXT-X-PROGRAM-DATE-TIME:2021-10-05T15:07:04.146+00:00')
[https @ 0x7fe921837600] Opening 'https://rr1---sn-hgn7rn7y.googlevideo.com/videoplayback/id/...
...
```

for every 5 seconds of downloaded playtime. So for a playlist containing many 1-hour videos a LOT of output! 

In addition, the download seems to be very slow. Every block for 5 second download of video takes about 5 second, so to dowenload a 1-hour video takes about one hour to download. 

Is that normal? Or am I missing something?

**Comment by coletdjnz** [comment]
I think you have a livestream in the playlist? Can you post the video in question and/or the playlist?

To skip livestreams I believe you can add `--match-filter "!is_live"`

**Comment by pukkandan** [comment]
Live streams download slowly and print a lot of output since it is using ffmpeg and not the native downloader. This is expected. Whether you are actually dealing with livestream, we dont know since you didn't show log

**Comment by alex4200** [comment]
Yes - there was a livestream in the playlist! When I skip it the rest of the videos are downloaded as expected. 

Thanks a lot for the help @coletdjnz  and @pukkandan
